templates/base.libsonnet

```diff
@@ -225,6 +226,7 @@ local volumes = import 'volumes.libsonnet';
         // Try 2 times before giving up.
         backoffLimit: 1,
         activeDeadlineSeconds: config.timeout,
+        ttlSecondsAfterFinished: timeouts.one_hour * 24 * 7,
```

Comment:
Can you put this in the `oneshotJob` template instead? CronJobs already have a cleanup policy.